week2/2-website/css/style.css

```diff
@@ -15,5 +42,199 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+img {
+  max-width: 100%;
+}
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  border-bottom: 1px solid #ddd;
+  width: 100%;
+}
+.logo {
+  width: 2.5rem;
+  margin-left: 15rem;
```

Comment:
Instead of margin left here, you could have added `margin: 0 auto;` to the container, or `.navbar` in this case.